securitycenter/src/main/java/com/google/cloud/examples/securitycenter/snippets/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,161 @@
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+/** Snippets for how to work with Notifications in Cloud SCC. */
+final class NotificationConfigSnippets {
+  private NotificationConfigSnippets() {}
+
+  public static NotificationConfig createNotificationConfig(
+      String organizationId, String pubsubTopic, String notificationConfigId) throws IOException {
+
+    // [START scc_create_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "config-id";
+    // String pubsubTopic = "projects/{your-project}/topics/{your-topic}";
+    // Ensure this ServiceAccount has the "pubsub.topics.setIamPolicy" permission on the topic.
+
+    String orgName = String.format("organizations/%s", organizationId);
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      CreateNotificationConfigRequest request =
+          CreateNotificationConfigRequest.newBuilder()
+              .setParent(orgName)
+              .setConfigId(notificationConfigId)
+              .setNotificationConfig(
+                  NotificationConfig.newBuilder()
+                      .setDescription("Java notification config")
+                      .setPubsubTopic(pubsubTopic)
+                      .setEventType(EventType.FINDING)
+                      .setStreamingConfig(
+                          StreamingConfig.newBuilder().setFilter("state = \"ACTIVE\"").build())
+                      .build())
+              .build();
+
+      NotificationConfig response = client.createNotificationConfig(request);
+      System.out.println(String.format("Notification config was created: %s", response));
+      return response;
+    }
+    // [END scc_create_notification_config]
+  }
+
+  public static boolean deleteNotificationConfig(String organizationId, String notificationConfigId)
+      throws IOException {
+
+    // [START scc_delete_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "config-id";
+
+    NotificationConfigName notificationConfigName =
+        NotificationConfigName.newBuilder()
+            .setOrganization(organizationId)
+            .setNotificationConfig(notificationConfigId)
+            .build();
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      client.deleteNotificationConfig(notificationConfigName);
+
+      System.out.println(String.format("Deleted Notification config: %s", notificationConfigName));
+    }
+    // [END scc_delete_notification_config]
+
+    return true;
+  }
+
+  public static NotificationConfig getNotificationConfig(
+      String organizationId, String notificationConfigId) throws IOException {
+
+    // [START scc_get_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "config-id";
```

Comment:
Where do I get this?  does it have a special format?   Should it be unique / what?

Reply:
I adjusted the inline comment to make it clear its user provided and unique.